packages/faucet/src/profile.ts

```diff
@@ -1,19 +1,25 @@
 import { pathToString } from "@cosmjs/crypto";
-import { makeCosmoshubPath, OfflineSigner, Secp256k1HdWallet } from "@cosmjs/launchpad";
+import { makeCosmoshubPath, Secp256k1HdWallet, SigningCosmosClient } from "@cosmjs/launchpad";
+import { DirectSecp256k1Wallet, isOfflineDirectSigner, OfflineSigner } from "@cosmjs/proto-signing";
+import { SigningStargateClient } from "@cosmjs/stargate";
+
+import * as constants from "./constants";
 
 export async function createWallets(
   mnemonic: string,
   addressPrefix: string,
   numberOfDistributors: number,
+  direct = true,
```

Comment:
Since we only use direct signing mode in stargate, I'd use a `stargate` flag here as well.